cockatrice/src/dlg_forgotpasswordreset.cpp

```diff
@@ -0,0 +1,101 @@
+#include <QLabel>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+#include <QDebug>
+
+#include "dlg_forgotpasswordreset.h"
+
+DlgForgotPasswordReset::DlgForgotPasswordReset(QString host, int port, QString playerName, QWidget *parent)
+    : QDialog(parent)
+{
+    hostLabel = new QLabel(tr("Host:"));
+    hostEdit = new QLineEdit(host);
+	hostLabel->hide();
```

Comment:
Nope.  The dialog's value is pulled to build the next command.  The dialog has line edits that are filled with data from the first response container of the server that was contacted to begin the process.  In a nutshell the entire process works like this:

client sends request to server for forgot password
server returns its host info and the value indicating its flagged the account for reset
client takes the response information and populates second dialog 
user fills in token (that was emailed) and new password sends it back to server
server resets user password

(keep in mind there are numerous checks that happen throughout the process)